pdal/util/FileUtils.cpp

```diff
@@ -234,7 +234,11 @@ std::vector<std::string> directoryList(const std::string& dir)
         fs::directory_iterator end;
         while (it != end)
         {
+#ifndef PDAL_WIN32_STL 
+            files.push_back(fromNative(it->path().string()));
+#else
             files.push_back(fromNative(it->path()));
```

Comment:
Afaict, implicit conversion from `std::filesystem::path` to `std::string` isn't defined in the standard.
It would probably good to replace this with
``` diff
-            files.push_back(fromNative(it->path()));
+            files.push_back(it->path().u8string());
```

That should also be working on all platforms. (No need to be conditional on `PDAL_WIN32_STL`.)